tomls/omnibus-arbitrum-mainnet/collaterals/susde.toml

```diff
@@ -0,0 +1,31 @@
+[var.susde_collateral_config]
+susde_max_collateral_limit = "<%= parseEther(String(3_000_000)) %>"
```

Comment:
can you please update to 10m https://sips.synthetix.io/sccp/sccp-354/ I understand not spelled out in the sccp, but the max amount on susde should be 10, on synth or the underlying

Reply:
done